colors/monochrome.vim

```diff
@@ -91,7 +91,7 @@ call s:hi('Folded')
 call s:hi('LineNr', s:dgray)
 
 " Small arrow used for tabs.
-call s:hi('SpecialKey', s:default_fg, s:default_bg, s:bold)
+call s:hi('SpecialKey', s:sblue, s:black, s:bold)
```

Comment:
This is what the comment was referring to

Reply:
Yes please, let's use `s:default_bg`.